src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,512 +1,4021 @@
 openapi: 3.0.0
 info:
-  description: X-Road Central Server Admin API. Note that the error metadata responses described in some endpoints are subjects to change and may be updated in upcoming versions.
-  version: "1.0.0"
-  title: X-Road Central Server Admin API
   contact:
+    email: info@niis.org
     name: Nordic Institute for Interoperability Solutions (NIIS)
     url: https://github.com/nordic-institute/X-Road-development/#enhancement-requests-and-error-reports
-    email: info@niis.org
+  description: X-Road Central Server Admin API. Note that the error metadata responses described in some endpoints are subjects to change and may be updated in upcoming versions.
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
+  title: X-Road Central Server Admin API
+  version: "1.0.0"
 servers:
-  - url: /api/v1
-    description: basepath for API v1
+  - description: basepath for API v1
+    url: /api/v1
+tags:
+  - name: backups
```

Comment:
Addressed in https://github.com/nordic-institute/X-Road/commit/68d52ca7b195e995c070a777f8d880c180bfb3c3

One exception: configuration-parts are after configuration-sources since they belong under that main level resource (but tag name is abbreviated)